packages/babel-generator/src/node/parentheses.ts

```diff
@@ -479,7 +479,10 @@ export function Identifier(
   // some tools (including earlier Babel versions) can't parse
   // `for await (async of [])` without them.
```

Comment:
Comments here seem no longer needed?

Reply:
Good catch! It looks this code was intentional, but the only mentioned tool I could find through issues + git blame was Babel, and the bug in Babel has been fixed long ago.